assets/src/data/model/status/helpers.js

```diff
@@ -213,6 +214,25 @@ const STATUS_TRANSLATION_MAP_DATETIME = {
 	),
 };
 
+/**
+ * Translation map for checkin statuses
+ *
+ * @type {{}}
+ */
+const STATUS_TRANSLATION_MAP_CHECKIN = {
+	[ CHECKIN_STATUS_ID.STATUS_CHECKED_IN ]: new Label(
+		__( 'check in', 'event_espresso' ),
+		__( 'checked in', 'event_espresso' )
+	),
+	[ CHECKIN_STATUS_ID.STATUS_CHECKED_OUT ]: new Label(
+		__( 'check out', 'event_espresso' ),
+		__( 'checked out', 'event_espresso' )
+	),
```

Comment:
The above two implementations are not strictly _singular_ and _plural_.  I thought it might be good to provide the singular version which is imperative and a "plural" version which is declarative.  However, I'm on the fence with this as it could be unexpected.  Perhaps plural versions should simply be `check ins` and `check outs`?

Reply:
ya I agree with your thoughts and it'd be nice to have some additional label type vos in the future if/when needed.  For now I'll revert what I did here.  As per further discussion in slack, I will also change the singular/plural versions  to be `check-in` and `check-out`.  Since the label is intended to be more of a "noun" representing the check in "status" we think that'd be more appropriate.